Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Execution_of_an_uncommon_process_at_an_early_startup_stage.yml

```diff
@@ -0,0 +1,1783 @@
+          - operator: uniq
+      FileRemediation:
+        simple: Quarantine
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+      forEach: true
+    view: |-
+      {
+        "position": {
+          "x": -9,
+          "y": 2540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "79":
+    id: "79"
+    taskid: 2ee2a177-42a7-4379-82d7-f515d6e1f666
+    type: collection
+    task:
+      id: 2ee2a177-42a7-4379-82d7-f515d6e1f666
+      version: -1
+      name: Manual Task - Analyst Decision for Next Actions
+      description: |-
+        Analyst approval is required for the following remediation actions:
+        - Quarantine malicious files
+        - Quarantine malicious files and add them to the block list
+        - Add malicious files to the block list
+        - No action
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "91"
+    scriptarguments:
+      key:
+        simple: Message
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2122
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: "**Select the response actions:**\nThe execution found as malicious, please select the response actions:\n\n--- \nMalicious Processes details:\n ---\n\n### **Action process details:**\n- **Action Process Path**:\n${MaliciousActionProcess.actionPath}\n - **Action Process Hash**:\n${MaliciousActionProcess.actionSha256}\n\n---\n\n### **Actor process details:**\n- ${.=val.MaliciousActorProcess ? \"**Actor process Path:** \" + val.MaliciousActorProcess.actorPath : \"N/A\"}  \n- ${.=val.MaliciousActorProcess ? \"**Actor process Hash:** \" + val.MaliciousActorProcess.actorSha256 : \"N/A\"}\n\n---\n\n### **CGO process details:**\n- ${.=val.MaliciousCGOProcess ? \"**CGO process Path:** \" + val.MaliciousCGOProcess.cgoPath : \"N/A\"}  \n- ${.=val.MaliciousCGOProcess ? \"**CGO process Hash:** \" + val.MaliciousCGOProcess.cgoSha256 : \"N/A\"}"
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - {}
+        - simple: Quarantine
+        - simple: Add to Block List
+        - simple: Quarantine and Add to Block List
+        - simple: No Action
```

Comment:
Might conflict with not choosing anything if the user is familiar with Else, which isn't handled in the playbook.

Reply:
else is not an option here.
![image](https://github.com/user-attachments/assets/961004d5-3fd1-419f-8b2b-d96e6c859a62)

Reply:
Yes. But what happens if the user doesn't choose anything and clicks submit?

Reply:
Done. the conditional task was updated